app/table/columns/action-col.tsx

```diff
@@ -66,7 +78,6 @@ type RowActionsProps = {
 export const RowActions = ({ id, copyIdLabel = 'Copy ID', actions }: RowActionsProps) => {
   return (
     <DropdownMenu.Root>
-      {/* TODO: This name should not suck; future us, make it so! */}
```

Comment:
This comment is from May 2022 #883 and refers to the aria label "Row actions", which tbh I don't see anything wrong with. That's what they are.